fili-core/src/test/groovy/com/yahoo/bard/webservice/web/ResponseSpec.groovy

```diff
@@ -107,7 +107,11 @@ class ResponseSpec extends Specification {
 
     Set<Column> columns
     Set<LogicalMetric> testLogicalMetrics
-    Response response
+    ResponseData response
```

Comment:
At a minimum, we should turn this into the `FiliResponseWriterSpec`, since that's what this is testing now.

However, we really should break this apart. Anything that tests only one format (i.e. just CSV, or just JSON) should be moved into a spec for the appropriate `ResponseWriter`. Anything that tests the getters in `ResponseData` should be moved into `ResponseDataSpec`.

In my mind, testing should be the following:
1. There should be dedicated `Spec` files for each writer, which test that they serialize the format correclty.
2. There should be a dedicated `Spec` that tests those `ResponseData` utility methods that are interesting.
3. There should be a `Spec` for `FiliResponseWriter` that ensures it selects the right writer (heh).